talkthroughai/src/app/(dashboard)/page.tsx

```
@@ -3,14 +3,20 @@ import { auth } from "@/lib/auth";
+
+  const data = await caller.hello({text:"Adarsha Server"});
+
   const session = await auth.api.getSession({
     headers: await headers(),
   });
   if(!session){
   redirect("/sign-in");
   }
+
+  return <p>{data.greeting}</p>
   return <HomeView />;
```

Biome (1.9.4)

[error] This code is unreachable

... because this statement will return from the function beforehand


(lint/correctness/noUnreachable)